vnext/Universal.SampleApp/App.xaml

```diff
@@ -2,6 +2,7 @@
     x:Class="WindowsSampleApp.App"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:local="using:WindowsSampleApp">
+    xmlns:local="using:WindowsSampleApp"
+    RequestedTheme="Light">
```

Comment:
>    RequestedTheme="Light" [](start = 0, length = 26)

why do you need this?

Reply:
Without that, the Sample app attempts to replicate the default system theme and I noticed that a lot of the examples (such as flyout) render white test on a white background when the theme is dark. This forces the sample app to always have the light theme.

Reply:
We should fix the examples so that they respect the app theme. Can you open a bug for that?

This change also "breaks" the AppTheme example:
https://github.com/microsoft/react-native-windows/blob/master/vnext/src/RNTester/ThemingExample.uwp.tsx

It's probably best to not have the sample app look terrible in dark theme, so for the time being, please add something to the ThemingExample that explains why it is not working (probably link to the github issue) and what they need to do if they want to try it out (delete RequestedTheme form App.xaml"

Reply:
[Bug](https://github.com/microsoft/react-native-windows/issues/2757)